<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastrarCarro</class>
 <widget class="QDialog" name="CadastrarCarro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Carros</string>
  </property>
  <widget class="QComboBox" name="cBoxMarca">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCadastrar">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PLACA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>COR:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MARCA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MODELO:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxUf">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxCidade">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>UF:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CIDADE:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePlaca">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>AAA9999</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Cor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMarca">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Marca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModelo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Modelo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCidade">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Cidade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUf">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add UF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>151</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="data.qrc">:/img/carro.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxCor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxModelo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cBoxUf</tabstop>
  <tabstop>cBoxCidade</tabstop>
  <tabstop>cBoxMarca</tabstop>
  <tabstop>pushCadastrar</tabstop>
 </tabstops>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushCadastrar</sender>
   <signal>clicked()</signal>
   <receiver>CadastrarCarro</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
